metadoc-js/src/main/scala/metadoc/MetadocAttributeService.scala

```diff
@@ -0,0 +1,40 @@
+package metadoc
+
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.Future
+import scala.meta._
+import scala.meta.internal.semantic.{schema => s}
```

Comment:
This one is "lower-level", it's the plain protobuf messages. For metadoc I've found the protobuf messages to be very useful, they match closely with the monaco api.